source/style-guide.erb

```diff
@@ -104,11 +104,49 @@ title: "Style guide"
 
   <div class="style-guide__container">
     <div class="style-guide__container-title">
-      Product story related tile
+      Product story quote with image
     </div>
     <div class="style-guide__container-body">
+      <%= partial 'product-stories/quote_caricature', locals: {
+        section: OpenStruct.new({
+          quote: "Consectetur adipisicing elit. Consequuntur sed aliquam vero repellendus impedit qui. Quia labore, quibusdam soluta, corrupti dolor deleniti voluptatum corporis quidem vero, beatae eius voluptatem quod!",
+          source: "Lorem ipsum",
+          image: "http://via.placeholder.com/150x254"
```

Comment:
We get mixed content warnings for these when viewing the preview site. Can we change them to `https`? (Same for the others below)